VEDA/django_secrets.py

```diff
@@ -19,6 +19,8 @@
         return_dict = None
 
 
+# Don't share with anybody.
+FERNET_KEYS = return_dict['fernet_keys'] or ['test-ferent-key']
```

Comment:
@mushtaqak can you confirm with @muhammad-ammar . I think we decided to remove django_secrets.py